### ISSUE TITLE
Thymeleaf-rails doesn't work with Rails 5.1.4

### ISSUE THREAD

**Issue by dhughes** [trabe/thymeleaf-rails#2]
I was attempting to use thymeleaf-rails with a Rails 5.1.4 project. I added this to my gemfile:

`gem 'thymeleaf-rails'`

However, when I ran `bundle install` I received this output:

```
The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x86-mswin32, x64-mingw32, java. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x86-mswin32 x64-mingw32 java`.
Fetching gem metadata from https://rubygems.org/.........
Fetching gem metadata from https://rubygems.org/..
Resolving dependencies...
Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 5.1.4)

  In Gemfile:
    rails (~> 5.1.4)

    thymeleaf-rails was resolved to 0.1.0, which depends on
      rails (>= 5.0.0.1, ~> 5.0.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

I'm new to Ruby and Rails so I'm not sure exactly what this is saying, but I suspect it has something to do with the versions of Rails that Thymeleaf-rails is declaring it can work with.

Is  Thymeleaf-rails known not to work with newer versions of Rails?

**Comment by davidbarral** [comment]
The gem should also work with Rails 5.1. I guess the restriction was there to force us to test it against the newer Rails releases. We'll fix the gemspec, test the gem against the latest rails versions and publish a new release.

**Comment by davidbarral** [comment]
Hi again!. Just published v0.1.1. It fixes the gemspec. Hope it works! :)
